scripts/debug_tools/failure_lib.py

```diff
@@ -40,6 +40,11 @@ def change_paths(string, pathModifierFun):
         # Note, this supports only POSIX paths.
         if string[i] == '/':
             path, path_end = find_path_end(string, i)
+            # Make sure that the prospective output folder exists.
+            if string[(i-3):(i-1)] == "-o":
```

Comment:
Is `-o/path/to/somewhere/a.o` valid? Or `-o` requires a space always after?
Also, `-o` could be followed by several whitespaces before the path.

Reply:
Sorry, that was a hotfix, I should have revised it. Replaced with regex matching.